tfhe/benches/integer/bench.rs

```diff
@@ -1030,11 +1283,40 @@ criterion_group!(
     unchecked_bitxor,
 );
 
+criterion_group!(
+    unchecked_parallelized_ops,
+    unchecked_abs_parallelized,
+    unchecked_add_parallelized,
+    unchecked_mul_parallelized,
+    unchecked_div_parallelized,
+    unchecked_rem_parallelized,
```

Comment:
btw @tmontaigu do we have a super fast rem/div for a scalar a power of 2 ?

div is a shift, rem is an and with a well chosen mask

Reply:
Yes there is